modules/ROOT/pages/CommunityResources/AcronymVault.adoc

```diff
@@ -0,0 +1,86 @@
+= Satisfactory Acronym Vault (SAV)
+
+This page exists to helps remember the excessive amount of acronyms used in our community.
+
+Originally a
+https://github.com/satisfactorymodding/SatisfactoryAcronymVault[GitHub repository of its own],
+it has been merged with the docs.
+
+Feel free to suggest changes on our Discord, or pull request them (Edit This Page in the top right), to help keep this page up to date.
+
+== In-Use Acronyms
+
+These acronyms are still actively in use.
+
+=== Coffee Stain Studios (CSS)
+
+_Not_ https://www.w3.org/Style/CSS/Overview.en.html[cascading style sheets].
+
+https://www.coffeestainstudios.com/[Coffee Stain Studios]
+is the Swedish game developer that created Satisfactory.
+
+=== Satisfactory Mod Loader (SML)
+
+This is the community-created mod responsible for loading other mods and providing an easy way for them to register content, configurations, and more.
```

Comment:
Does SML load mods anymore? I thought it didn't

Reply:
Fixed in 1dcbcda